client/my-sites/pages/page-card-info/index.jsx

```diff
@@ -53,13 +54,14 @@ function PageCardInfo( {
 	const moment = useLocalizedMoment();
 
 	const renderTimestamp = function() {
-		if ( page.status === 'future' ) {
+		if ( showPublishedStatus ) {
```

Comment:
This is `false` on the scheduled page filter view.

Reply:
Probably not the cleanest (we can tidy further later), but the following should fix the regression and keep search working:

```jsx
if ( page.status === 'future' || showPublishedStatus ) {
```

<img width="1113" alt="Screen Shot 2020-03-27 at 3 22 26 PM" src="https://user-images.githubusercontent.com/1270189/77805339-e033dd00-703e-11ea-9dac-c9f9a1e90f33.png">
<img width="1119" alt="Screen Shot 2020-03-27 at 3 22 35 PM" src="https://user-images.githubusercontent.com/1270189/77805349-e629be00-703e-11ea-9767-5966badacd3b.png">

Reply:
Oh, I am not sure how I missed that!  I guess I somehow didn't realize that `PostRelativeTimeStatus` was responsible for treating the date differently for scheduled posts.  Yeah, that looks like a good fix.